sites/docs/src/content/docs/guidelines/components/modules.md

```diff
@@ -464,7 +464,30 @@ tuple val(meta), path('*.tab'), emit: tab,  optional: true
 
 ### One output channel per output file type
 
-Each output file SHOULD be emitted in its own channel (and no more than one), along with the `meta` map if provided ( the exception is the versions.yml ).
+Each output file type SHOULD be emitted in its own channel (and no more than one), along with the `meta` map if provided ( the exception is the versions.yml ).
+
+In some cases the file format can be different between files of the same type (e.g. indices: `.bai` and `.crai`). These different file formats SHOULD be part of the same output channel since they are the same file type.
```

Comment:
You should sort of specify that they can go in the same channel, as they 1) serve the same purpose and 2) the two formats are mutually exclusive.

Reply:
Done!